src/page-about/AboutPage.tsx

```diff
@@ -12,69 +16,278 @@ const AboutPage: React.FC = () => (
+              Can I make changes to this model, or re-share it with others?
+            </h2>
+            <p className="leading-7 text-base sm:text-lg mb-6 font-body">
+              You can modify and re-share this model, so long as you keep the
+              license notice in the Terms of Service unchanged. It ensures that
+              others you pass the model along to also have the ability to modify
+              and re-share.
+            </p>
+            <h2 className="text-xl text-left font-display font-normal mt-6 mb-3">
+              What other guidelines or recommendations are available to prepare
+              our system for COVID-19?
+            </h2>
+            <ul className="list-disc list-inside md:list-outside text-base sm:text-lg font-body leading-normal">
+              <li className="my-2">BOP: COVID-19 Resource Page </li>
+              <li className="my-2">AJA: COVID-19 Resources </li>
+              <li className="my-2">
+                CDC: Guidance on Management of Coronavirus (COVID-19) in
+                Correctional and Detention Facilities
+              </li>
+            </ul>
+            <p className="leading-7 text-base sm:text-lg mb-6 font-body"></p>
+            <h2 className="text-xl text-left font-display font-normal mt-6 mb-3">
+              How can I help other criminal justice systems?
+            </h2>
+            <p className="leading-7 text-base sm:text-lg mb-6 font-body">
+              The best way to help the rest of the community in real-time is to
+              join the 'Criminal Justice C19' community on Slack (see{" "}
+              <a href="https://www.recidiviz.org/covid">
+                https://www.recidiviz.org/covid
+              </a>
+              ). This is a community of criminal justice department leads and
+              research staff sharing knowledge on how they're approaching the
+              challenges posed by COVID-19.
+            </p>
+            <h1 className="text-2xl sm:text-3xl text-left font-display mt-10">
+              Contact us
+            </h1>
             <p className="leading-7 text-base sm:text-lg my-6 font-body">
-              Recidiviz, in collaboration with X and Y, is standardizing and
-              tracking those actions that state and county leaders across the
-              country have taken to reduce populations and transmission rate.
-              Why? Because we believe in transparency, and that knowing what’s
-              happening where is helpful to those empowered to take action. In
-              the coming days, we’ll add more data sources to this site intended
-              to aid decision-makers understand what else they can do and what
-              impact those decisions will have both now and in the long run for
-              public health and criminal justice alike.
+              Questions? Feedback? Need help refining this model or fitting it
+              to your system? Email to{" "}
```

Comment:
"Email to covid@recidiviz.org..." reads a bit strangely. Let's change it to "Reach out to..."